core/src/main/java/org/godotengine/plugin/gast/GastNode.kt

```diff
@@ -58,6 +63,50 @@ class GastNode @JvmOverloads constructor(
         gastManager.registerGastRenderListener(this)
     }
 
+    fun getProjectionMesh() : ProjectionMesh {
+        checkIfReleased();
+        val meshPointer = nativeGetProjectionMesh(nodePointer)
+
+        if (projectionMeshPool.containsKey(meshPointer)) {
+            return projectionMeshPool.getValue(meshPointer)
+        }
+
+        when(getProjectionMeshType()) {
+            ProjectionMeshType.RECTANGULAR ->
+                projectionMeshPool[meshPointer] =
+                    RectangularProjectionMesh(meshPointer, nodePointer)
+            ProjectionMeshType.EQUIRECTANGULAR ->
+                projectionMeshPool[meshPointer] =
+                    EquirectangularProjectionMesh(meshPointer, nodePointer)
+            else -> {
+                TODO("Mesh type unimplemented")
+            }
+        }
+
+        return projectionMeshPool.getValue(meshPointer)
+    }
+
+    private external fun nativeGetProjectionMesh(nodePointer: Long) : Long
+
+    fun getProjectionMeshType() : ProjectionMeshType {
+        checkIfReleased()
+        return ProjectionMeshType.values()[nativeGetProjectionMeshType(nodePointer)]
+    }
+
+    private external fun nativeGetProjectionMeshType(nodePointer: Long) : Int
+
+    fun setProjectionMesh(projectionMeshType: ProjectionMeshType) {
+        checkIfReleased()
+        setProjectionMesh(projectionMeshType.ordinal)
+    }
+
+    private fun setProjectionMesh(projectionMeshType: Int) {
+        checkIfReleased()
+        nativeSetProjectionMesh(nodePointer, projectionMeshType)
+    }
```

Comment:
Seems this method can just be merged into its caller above.

Reply:
Done.